vectordatasource/util.py

```diff
@@ -75,3 +75,21 @@ def calculate_path_major_route(fid):
 def cycling_network(tags, fid):
     # TODO: implement me! current implementation is a stub.
     return None
+
+
+def tag_yes_to_bool(str_or_none):
+    return True if str_or_none == 'yes' else None
```

Comment:
Ooops, I should have read further down below. According to taginfo, there are no `true` values for `capital` or `state_capital`, so we can probably just leave this as is.